test/e2e/BrsComponents.test.js

```diff
@@ -117,7 +117,7 @@ describe("end to end brightscript functions", () => {
             "Seconds: ",
             "15",
             "Last Day of Month: ",
-            "30",
+            "29",
```

Comment:
Why did this need to change?

Reply:
the tests were failing

Reply:
<img width="489" alt="Screenshot 2023-10-24 at 08 34 19" src="https://github.com/rokucommunity/brs/assets/35704032/8ef4e72d-003b-45ac-97c5-63ec7c9e3a42">
they are failing on master as well